sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs

```diff
@@ -333,10 +368,10 @@ async Task<bool> OnBreakpointHit (SessionId sessionId, JObject args, Cancellatio
 				//Give up and send the original call stack
 				return false;
 			}
-			var bp = context.Breakpoints.FirstOrDefault (b => b.RemoteId == bp_id.Value);
 
-			var store = context.Store;
-			var src = bp == null ? null : store.GetFileById (bp.Location.Id);
+			var bp = context.BreakpointRequests.Values.SelectMany (v => v.Locations).FirstOrDefault (b => b.RemoteId == bp_id.Value);
+
+			var src = bp == null ? null : (await LoadStore (sessionId, token)).GetFileById (bp.Location.Id);
```

Comment:
`src` is not being used anywhere. Is it just to verify that we have a file corresponding to that?
Maybe we should:

```
var store = await LoadStore (sessionId, token);
// and then check GetFileById, if we want that
```

.. and remove the call to `LoadStore` inside the loop, since IIUC, this shouldn't change for the same `sessionId`.

Reply:
removed